corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -78,3 +78,8 @@
     href="{% static '@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css' %}"
     type="text/css"
   />
+  <link
+    rel="stylesheet"
+    href="{%static 'cloudcare/css/markdown-table.css' %}"
```

Comment:
Optional: This would be nicer as a LESS file, since you could take advantage of nesting which I think makes styling files more readable. Side benefit, it would also make stickler stop complaining about rules not being alphabetical.